_infra/helm/collection-instrument/values.yaml

```diff
@@ -69,3 +69,6 @@ gcp:
   topic: topic
 
 saveSeftInGcp: false
+
+gnu_recipient: sdx_preprod@ons.gov.uk
```

Comment:
Rather an underscores, these should be in camelCase to match the other vars, i.e. gnuRecipient 

or actually even better you could do:
```
gnu:
  recipient: sdx_preprod@ons.gov.uk
  fingerprint: A8F49D6EE2DE17F03ACF11A9BF16B2EB4D779991
```

then in the values.yaml

```
 - name: ONS_GNU_FINGERPRINT            value: "{{ .Values.gnu.fingerprint }}"          
 - name: ONS_GNU_RECIPIENT            value: "{{ .Values.gnu.recipient }}"
```